### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by alpiyev99** [webstudio-is/webstudio-community#86] [Q&A]
![Снимок экрана 2023-12-10 в 05 40 09](https://github.com/webstudio-is/webstudio-community/assets/124505479/64439de9-376b-414a-973d-9db37c0f5402)
Where is positioning?

**Comment by kof** [comment]
Doesn't seem like a bug. You want an absolute position? like top/right/bottom/left?

**Comment by alpiyev99** [comment]
no, layout
[image: Снимок экрана 2023-12-11 в 02.02.55.png]

вс, 10 дек. 2023 г. в 23:11, Oleg Isonen ***@***.***>:

**Comment by kof** [comment]
Can you explain the problem with a full sentence please? I don't understand the question.

**Comment by alpiyev99** [comment]
You are webflow analog, why do you not provide Layouts? I want to see
experience like webflow.

пн, 11 дек. 2023 г. в 02:24, Oleg Isonen ***@***.***>:

**Reply by kof** [reply]
I still don't understand what you mean, flex is there, grid is not YET implemented

**Comment by alpiyev99** [comment]
I belive in you, lets do it man

пн, 11 дек. 2023 г. в 16:25, Oleg Isonen ***@***.***>:
